docs/kubernetes/operator/instrumenting-python.md

```diff
@@ -0,0 +1,179 @@
+                value: "debug"
+              - name: AGENT_HAS_STARTED_IF_YOU_SEE
+                value: "Exception while exporting metrics HTTPConnectionPool"
+    EOF
+    ```
+
+
+2. Enable auto-instrumentation of the Python application using one of the following methods:
+
+  - Add an annotation at namespace level (this will make all Pods of the namespace to be instrumented):
+
+    ```bash
+    kubectl annotate namespace python instrumentation.opentelemetry.io/inject-python=opentelemetry-operator-system/elastic-instrumentation
+    ```
+
+  - Alternatively, edit the `python-app` deployment to include the annotation under `spec.template.metadata.annotations`:
+
+    ```yaml
+    spec:
+    ...
+      template:
+        metadata:
+          labels:
+            app: python-app
+          annotations:
+            instrumentation.opentelemetry.io/inject-python: opentelemetry-operator-system/elastic-instrumentation
+    ...
+    ```
+
+3. Restart application:
+
+  Once the annotation has been set, the Pods need to be recreated for the instrumentation libraries to be injected.
+
+    ```bash
+    kubectl rollout restart deployment python-app -n python
+    ```
+
+4. Verify the auto-instrumentation resources are injected in the Pod:
+
+  Python apps are instrumented by the OpenTelemetry Operator with the following actions:
+
+  - It adds an init container in the Pod with the objective of copying the SDK to a shared volume.
+
+  - Defines an `emptyDir volume` mounted in both containers.
+
+  - Adds `PYTHONPATH` and other OTEL related environment variables.
+
+  Run `kubectl describe pod python-app-xxxx` and check:
+
+  - There should be an init container named `opentelemetry-auto-instrumentation-python` in the Pod:
+
+    ```bash
+    $ kubectl describe pod python-app-8d84c47b8-8h5z2 -n python
+    ...
+    ...
+    Init Containers:
+      opentelemetry-auto-instrumentation-python:
+        Container ID:  containerd://fdc86b3191e34ef5ec872853b14a950d0af1e36b0bc207f3d59bd50dd3caafe9
+        Image:         docker.elastic.co/observability/elastic-otel-python:0.3.0
+        Image ID:      docker.elastic.co/observability/elastic-otel-python@sha256:de7b5cce7514a10081a00820a05097931190567ec6e18a384ff7c148bad0695e
+        Port:          <none>
+        Host Port:     <none>
+        Command:
+          cp
+          -r
+          /autoinstrumentation/.
+          /otel-auto-instrumentation-python
+        State:          Terminated
+          Reason:       Completed
+    ...
+    ```
+
+  - The main container has new environment variables: 
+
+    ```bash
+    ...
+    Containers:
+      python-app:
+    ...
+        Environment:
+    ...
+          PYTHONPATH:                          /otel-auto-instrumentation-python/opentelemetry/instrumentation/auto_instrumentation:/otel-auto-instrumentation-python
+          OTEL_EXPORTER_OTLP_PROTOCOL:         http/protobuf
+          OTEL_TRACES_EXPORTER:                otlp
```

Comment:
0.113.0 would have also `OTEL_LOGS_EXPORTER: otlp` set

Reply:
The reported output is suppoed to be an example that might not match `exactly` the user's outcome. Do you think I should clarify that in the doc to avoid confusion?

Do you want me to add the `OTEL_LOGS_EXPORTER: otlp` even if as of today we are still not using `0.113.0` operator?

Reply:
I think it's fine as is